great_expectations/render/renderer/page_renderer.py

```diff
@@ -383,6 +384,7 @@ def _render_validation_info(cls, validation_results):
                     ["Great Expectations Version", ge_version],
                     ["Run Name", run_name],
                     ["Run Time", run_time],
+                    ["Run Duration", run_duration],
```

Comment:
Could we add the unit of time here as well? Currently it's just the number, so it might not be clear exactly how long the run_duration actually was.